doc/introduction/interfaces/torch.rst

```diff
@@ -185,8 +185,8 @@ we can do the following:
     def cost(phi, theta):
         return torch.abs(circuit4(phi, theta) - 0.5)**2
 
-    phi = torch.tensor([0.011, 0.012], requires_grad=True)
-    theta = torch.tensor(0.05, requires_grad=True)
+    phi = torch.tensor([0.011, 0.012], requires_grad=True, device='cuda')
+    theta = torch.tensor(0.05, requires_grad=True, device='cuda')
```

Comment:
@antalszava I don't think we should use `device='cuda'` explicitly in the documentation, since this will fail for users that don't have CUDA installed or GPU access. For example, if I run this snippet locally, I get:

```python
AssertionError: Torch not compiled with CUDA enabled
```

Reply:
Good point! Made the change as the text states

> with the classical nodes processed on a GPU

and the output was not put on CUDA devices. Maybe having

```python
    dev = "cuda"  torch.cuda.is_available() else "cpu"
    phi = torch.tensor([0.011, 0.012], requires_grad=True, device=dev)
    theta = torch.tensor(0.05, requires_grad=True, device=dev)
```
could help that the example is executable even in such cases.

Reply:
🤔 This is a case where I would probably suggest removing all references to the GPU in this page altogether. Users can refer to PyTorch docs for using GPUs, but we shouldn't have code examples in our docs that not all users can run